README.md

```diff
@@ -1 +1,156 @@
-# Hello_Microverse
\ No newline at end of file
+<a name="Hello Microverse"></a>
+
+<div align="center">
+  
+
+
+  <h3><b>Hello Microverse</b></h3>
+
+</div> 
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+> This is my portfolio I build it with html and css.
+
+**[Hello Microverse]** is the first project I officially created on day 2 of my first week studying software programming.
+
+## 🛠 Built With <a name="built-with"></a>
+
+1. html
+2. css
+3. Linters
+
+### Tech Stack <a name="tech-stack"></a>
+
+1. html
+2. css
+3. Linters
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://indexhtml.org/">HTML</a></li>
+    <li><a href="https://stylecss/">CSS</a></li>
+    <li><a href="https://linters.org/">Linters</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+index.html file.
+style.css file.
+Node.js modules
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Web browser
+Code editor
+
+### Setup
+
+Clone this repository to your desired folder:
+Run this command:
+git clone https://github.com/Mike47ip/Hello_Microverse_Project.git
+
+### Install
+
+Install this project with:
+
+cd my-project
+npm install
```

Comment:
:red_circle: I don't think you need to use the npm install command to run your project. You can remove it.